source/FreeRTOS_TCP_State_Handling.c

```diff
@@ -990,8 +990,7 @@
         #endif /* ipconfigSUPPORT_SELECT_FUNCTION */
 
         /* And bind it to the same local port as its parent. */
-        /*TODO xAddress.sin_address.ulIP_IPv4 = *ipLOCAL_IP_ADDRESS_POINTER; */
-        xAddress.sin_port = FreeRTOS_htons( pxSocket->usLocalPort );
+        FreeRTOS_GetLocalAddress( pxSocket, &xAddress );
```

Comment:
Please make it MISRA compliant by making use of the return value. 

` The return value of a non-void function "FreeRTOS_GetLocalAddress" is unused.`